examples/nextjs-lit/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "nextjs-lit",
```

```suggestion
  "name": "@lit-internal/examples-nextjs",
```

For security we should avoid ever having packages, even marked internal, that we don't own (or own the namespace for, so that nobody else can take it).

---

Done!